docker-images/kafka/cruise-control-scripts/cruise_control_config_generator.sh

```diff
@@ -42,6 +42,7 @@ capacity.config.file=$CC_CAPACITY_FILE
 cluster.configs.file=$CC_CLUSTER_CONFIG_FILE
 webserver.accesslog.path=$CC_ACCESS_LOG
 webserver.http.address=0.0.0.0
+webserver.http.cors.allowmethods=OPTIONS,GET
```

Comment:
Why does this need to be there by default? How does CC handle it when the users configure the same?

Reply:
We want to grant read-only access to the CC API from other origins but only read-only access. This configuration option is forbidden so users cannot give themselves write access to the CC API by adding the "POST" value to this configuration option. It will be ignored if a user tries to configure this option